addon/components/horizontal-bar-chart.js

```diff
@@ -367,10 +359,85 @@ export default ChartComponent.extend(FloatingTooltipMixin,
     return this.get('yAxis').attr(this.get('axisAttrs'));
   },
 
+  /**
+   * Given the list of elements for the group labels and value labels,
+   * determine the width of the largest label on either side of the chart.
```

Comment:
This function seems not return the width. It seems to return left & right coordinate of the label